src/website/src/pages/ec/components/button/docs/usage.mdx

```diff
@@ -9,33 +9,54 @@ import { Paragraph } from '@ecl/website-components';
-| Call-to-action button | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_CTA_Default.png)       | for the most important action on the page like "Apply for funding" or "Watch live streaming". |
-| Primary button        | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Primary_Default.png)   | for primary actions in forms, to initiate a search query or to apply a filter.                |
-| Secondary button      | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Secondary_Default.png) | or secondary actions.                                                                         |
-| Ghost button          | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Ghost_Default.png)     | when an action does not require dominance on a page, for instance to "cancel" a form.         |
-| Search button         | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Search_Default.png)    | use exclusively for the search function.                                                      |
+ECL has five buttons with different visual designs to make a distinction between the kind of action the button will carry out.
+The button with the strongest visual weight will attract the most attention.
 
-Each page can have one to two Primary (or Call-to-action) buttons. Any remaining calls-to-action should be represented as Secondary or Ghost buttons.
+| Button name                 | Visual style                                                                                            | Usage                                                                                                                                                                                               |
+| --------------------------- | ------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Call-to-action (CTA) button | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_CTA_Default.png)       | funnel users into the most important action for your business ex: "Apply for funding" or "Watch live streaming". Limit to one CTA on a given page, the same CTA can be repeated within in the page. |
+| Primary button              | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Primary_Default.png)   | for primary actions in forms, ex: "Log in", "Get funding", "Register", "Submit a request", should appear only few occurence on a page, ex: only 1 primary button per form                           |
```

Comment:
this is the only button where we talk about forms. All other buttons are related to page.